plugins/catalog/api-report.md

```diff
@@ -215,6 +215,8 @@ export interface DefaultCatalogPageProps {
   // (undocumented)
   emptyContent?: ReactNode;
   // (undocumented)
+  filters?: ReactNode;
```

Comment:
Hmm, thoughts on making this an internal API for now? We've been resisting similar changes to this component because we want to be able to change the default catalog page quite freely. Then again this is at a pretty good level of abstraction though, thoughts @backstage/catalog-maintainers, @timbonicus?

Reply:
I extracted a base catalog page layout component to avoid changing the default catalog page props, does it look good to you 🙂 ? Here is the commit: https://github.com/backstage/backstage/pull/20355/commits/dfd0b8eec5d6ad190efdc9d2934a189f58f0d251.